build.gradle

```diff
@@ -44,6 +48,9 @@ buildscript {
 
         classpath 'com.google.protobuf:protobuf-gradle-plugin:0.8.12'
 
+        // Use the Glean Gradle plugin.
+        classpath "org.mozilla.components:tooling-glean-gradle:$android_components_version"
```

Comment:
I guess I didn't realize this would apply to other `build.gradle` files, but given that this is working, it must.  (Our docs sort of assume it's all together in the same `build.gradle` and maybe there's an update we should make there...)

Reply:
Maybe this is also due to the way A-S has split the Gradle files: in all the projects we integrated with the root build.gradle contained the majority of the directives. It is not the same here :)